tap_mambu/helpers/client.py

```diff
@@ -118,13 +119,15 @@ def __init__(self,
                  subdomain,
                  apikey_audit,
                  page_size,
-                 user_agent=''):
+                 user_agent='',
+                 window_size=1):
         self.__username = username
         self.__password = password
         self.__subdomain = subdomain
         base_url = "https://{}.mambu.com/api".format(subdomain)
         self.base_url = base_url
         self.page_size = page_size
+        self.window_size=window_size
```

Comment:
Maintain the spacing as on previous lines

Reply:
Addressed already.